### ISSUE TITLE
Factor to adjust available assessment rates to "market" value

### ISSUE THREAD

**Issue by dcaud** [whoownsphilly/wop-web#156]
TL;DR: My gut says that the assessed values may be close to the market values, so an adjustment factor doesn't help. However, my gut feels rather uninformed and more work may be fruitful in this regard. Instead of a factor, using some like values from Zillow may be better, but I'm not sure that is worth the effort.

The city makes available each property's assessed value. These assessed values may be different (and presumably often lower) than the value of the property at sale. There was a discussion that using a factor to adjust the assessed values could be used to create the value shown on WhoOwnsPhilly.org. My initial research is showing that 1) the assessed value is an attempt to be close to market value; and 2) there are adjustment factors (called Common Level Ratios) made available by the State for each county, however, the factor for Philly suggests that the assessed values are rather close to market values. An [Inquirer article}(https://www.inquirer.com/news/assessment-appeal-montco-chester-bucks-delco-delaware-county-reassessment-20190729.html) discusses the Philly assessment vs. market value issue. 

My quick review of the map [here](https://www.inquirer.com/philly/news/philadelphia-property-assessments-appeal-tax-market-value-20180810.html) suggests that many of the poorest neighborhoods have over-assessed property values, which would mean poorer neighborhoods disproportionately pay higher taxes.

Here's some more citations:

- The Philly county Common Level Ratio for 2020 was 1.07. I believe this means that a property with an assessed value of 100,000 has a "market" value of close to 107,000 (100,000 * 1.07): https://www.revenue.pa.gov/TaxTypes/RTT/Pages/Common%20Level%20Ratios.aspx

- Info on CLR with examples: https://www.suburbanrealtorsalliance.com/blog/2019/07/26/local-issues/what-is-an-accurate-property-assessment/

- General info on CLR: https://revenue-pa.custhelp.com/app/answers/detail/a_id/229/kw/Common%20Level%20Ratio/session/L3RpbWUvMTY0MTA4NjQ2NS9zaWQvMm9tMV80dXA%3D
